src/EditorFeatures/CSharpTest/PerfTests/CSharpPerfFindRefs.xml

```diff
@@ -24,8 +24,8 @@
       <WaitForCodeMarkers />
       <ExecuteCommand Command="Window.ActivateDocumentWindow" />
 
-      <VerifyRoslynModulesLoadedStatus ExpectedStatus="CSharp" />
-      <VerifyRoslynModulesLoadedStatus ExpectedStatus="NoBasic" />
+      <!--<VerifyRoslynModulesLoadedStatus ExpectedStatus="CSharp" />
```

Comment:
Well, this change isn't all the tests. I left alone the typing tests and the diagnostics tests.